recipes/treematching/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "treematching" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/LionelAuroux/{{ name }}/archive/master.tar.gz
```

Comment:
We really need some kind of release to tie this to.

Reply:
Installing from master is a blocker for this, we cannot package this as such.